### PR TITLE
kar de bc

### PR DESCRIPTION
**Pull request by akit01**
l lelo

### CODE REVIEW

File: branch_practice_project/src/branch_practice_project/Main.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 /**
  * @author nnikhil
  * Nikhil added it on head.
+ * Nikhl ne fir add kar dia.
```
Comment (nikhilkmr) [inline branch_practice_project/src/branch_practice_project/Main.java]: ye kya badtamiz comment he ?

File: branch_practice_project/src/branch_practice_project/User.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package branch_practice_project;
+
+public class User
```
Comment (nikhilkmr) [inline branch_practice_project/src/branch_practice_project/User.java:3]: change class name.

**Review by nikhilkmr — CHANGES_REQUESTED** [review]
aise badde comment ?

File: branch_practice_project/src/branch_practice_project/User.java (hunk 2 above)
Comment (akit01) [reply branch_practice_project/src/branch_practice_project/User.java:3]: Done

**Review by nikhilkmr — APPROVED** [review]

**Comment by nikhilkmr** [comment]
sab kuch changa.

**Pull request merged**